pkg/agent/flannel/flannel.go

```diff
@@ -132,6 +148,14 @@ func WriteSubnetFile(path string, nw ip.IP4Net, ipMasq bool, bn backend.Network)
 
 	fmt.Fprintf(f, "FLANNEL_NETWORK=%s\n", nw)
 	fmt.Fprintf(f, "FLANNEL_SUBNET=%s\n", sn)
+
+	if nwv6.String() != emptyIPv6Network {
+		snv6 := bn.Lease().IPv6Subnet
+		snv6.IP = (*ip.IP6)(big.NewInt(0).Add((*big.Int)(snv6.IP), big.NewInt(1)))
```

Comment:
This seems like a long way to go to emulate the concise `sn.IP++` that we have for IPv4 a couple lines earlier. Are there no helpers available to make this cleaner for the commensurate IPv6 operations? Should we consider writing them? Stated another way: IPv4 arithmetic is obviously different than IPv6 arithmetic but the use of "math/big" sure does make it clunky.

Reply:
Yes, I already spotted this. I know it is a bad excuse but I just copied what was done upstream. Let me investigate a bit more but at least we should consider hiding this behind a library, possibly in flannel upstream

Reply:
This should be done